sections/categories.liquid

```diff
@@ -0,0 +1,43 @@
+{{ 'categories.css' | asset_url | stylesheet_tag }}
+<div class="categories__container">
+  <div class="categories__wrapper">
+    {% for block in section.blocks %}
+      <div class="category">
+        <div class="category__image">
+          <img
+            src="{{ block.settings.collection.image | img_url: 'master' }}"
```

Comment:
If you already know the image size, you can pass it to the image filter. Also, `img_url` is deprecated

https://shopify.dev/docs/api/liquid/filters/img_url

```suggestion
            src="{{ block.settings.collection.image | image_url: width: 294, height: 154 }}"
```